modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -1168,6 +1202,22 @@ def date_is_valid?(date, property, is_full_date = false) # rubocop:disable Style
         false
       end
 
+      def address_is_military?(addr)
+        city = addr['city']
+        state = addr['state']
+
+        return true if MILITARY_CITY_CODES.include?(city)
+        return true if MILITARY_STATE_CODES.include?(state)
+
+        false
+      end
```

Comment:
How about:

```suggestion
      def address_is_military?(addr)
        MILITARY_CITY_CODES.include?(addr['city']) || MILITARY_STATE_CODES.include?(addr['state'])
      end
```
?

Reply:
Also, should we `upcase` the city and state as we do in the EVSS mapper?

Reply:
sharing the methods addresses this now